lib/lambda-worker/lambda-worker.ts

```diff
@@ -45,6 +47,12 @@ export class LambdaWorker extends cdk.Construct {
       );
     }
 
+    if (!this.isContainerLambda(props) && !this.isFunctionLambda(props)) {
+      throw new Error(
+        `Invalid lambdaProps only dockerImageTag/ecrRepositoryArn/ecrRepositoryName or handler/entry can be specified.`
+      );
+    }
```

Comment:
Hmm I think this check would throw when neither is specified, which I think we need such check. But another check is needed to ensure we don't have both set.

Reply:
There is a test case for all being set and ensuring that the error is thrown, see the array/table of tests that are ran for these options.

Reply:
Ah sorry, I commented before looking at `isContainerLambda` and `isFunctionLambda` - all good here.